packages/suite/src/views/wallet/receive/components/CoinjoinReceiveWarning.tsx

```diff
@@ -35,6 +35,7 @@ const InfoIcon = styled(Icon)`
 `;
 
 const Text = styled.div`
+    padding-right: 12px;
```

Comment:
I think this is not needed if you allow `gap` on the container for all screens.

Reply:
5b0ca0c1a